keycast.el

```diff
@@ -290,16 +290,16 @@ instead."
         (unless cons
           (setq keycast-mode nil)
           (user-error
-           "Cannot turn on %s.  %s not found in %s.  Try customizing %s."
+           "Cannot turn on %s.  %s not found in %s.  Try customizing %s"
```

Comment:
Generally I follow the convention of not ending messages with a period, but it feels weird for multi-sentence message and leaves me wondering if the message was somehow truncated. So I don't do it in those cases.

Reply:
I don't now either. I usually just follow the conventions, so far without bad side effects. If the message is truncate the trailing period is so or so removed?

Reply:
I mean the non-truncated message that lacks the period looks like it might be truncated (even though it is not).

Reply:
Hmm, right. I usually try to avoid writing multiple sentences to mitigate this problem ;)